homeassistant/components/totalconnect/alarm_control_panel.py

```diff
@@ -81,7 +81,7 @@ def __init__(
         for most users with new support for partitions.
         Add _# for partition 2 and beyond.
         """
-        if partition_id == 1:
+        if partition_id == "1":
             self._attr_name = None
             self._attr_unique_id = str(location.location_id)
         else:
```

Comment:
you can remove the `str()` down below

Reply:
I forced it to an `int` so now the `str()` is still needed